Source/Charts/Renderers/Renderer.swift

```diff
@@ -12,21 +12,23 @@
 import Foundation
 import CoreGraphics
 
+//@objc(ChartRenderer)
```

Comment:
what is this, commented out code or documentation? If it's doc, maybe use doc style comment like /* or other

Reply:
That shouldn't be commented.